src/res_printer.ml

```diff
@@ -462,16 +462,23 @@ let printPolyVarIdent txt =
     | NormalIdent -> Doc.text txt
 
 
-let printLident l = match l with
+let printLident l =
+  let rec hasLapply = function
+  | Longident.Lident _ -> false
+  | Longident.Ldot (path, _) -> hasLapply path
+  | Longident.Lapply _ -> true
+  in
+  match l with
   | Longident.Lident txt -> printIdentLike txt
-  | Longident.Ldot (path, txt) ->
+  | Longident.Ldot (path, txt) when not (hasLapply path) ->
```

Comment:
Is there a way to not have to "pay" perf wise for `Lapply`? The ReScript parser doesn't parse it at the moment.

Reply:
I'm not sure, maybe we can just ignore it since it's only happening when parsing from reason / ml.